requirements.txt

```diff
@@ -1,3 +1 @@
-numpy~=1.20.1
-scipy~=1.4.1
-cirq==0.10.0.dev20201203012958
+cirq~=0.10.0
```

Comment:
I was also tempted to remove numpy and scipy (since they are implicitly imported by cirq), but it looks like this is not a good practice:
https://github.com/unitaryfund/mitiq/issues/508

Reply:
I don't understand and/or disagree with the argument there. Why are scipy and numpy specifically pulled out from the Cirq requirements? We also use, e.g., matplotlib, but that's not pulled out from the Cirq requirements.

Reply:
The argument for explicitly listing anything we also directly need is in case Cirq or what ever other package we are relying on changes their dependencies, then we would be missing it and it would break our config.